pywbem/cim_operations.py

```diff
@@ -453,11 +460,22 @@ class WBEMConnection(object):
         :class:`~pywbem.BaseOperationRecorder`, each operation
         that is executed on this connection will be recorded by
         invoking its :meth:`~pywbem.BaseOperationRecorder.record` method.
+
+      use_pull_operations (:class:`py:bool`):
+        Indicates whether the client will attempt the use of pull operations in
+        any `Iter...()` methods.
+
+        This variable is initially set from the same-named input argument, and
+        is modified when `Iter...()` methods are invoked, dependent on the
+        support for pull operations in the WBEM server.
+        After modifying the variable, it is either `True` or `False` and
+        indicates whether pull operations will be used in any subsequent
+        `Iter...()` methods.
```

Comment:
In order to reflect the fact that only for an initial value of `None`, the instance variable will be adjusted, I suggest to replace the paragraph above with this:

```
        This variable is initially set from the same-named input argument. If
        it is `None`, it is adjusted when `Iter...()` methods are invoked, dependent
        on the support for pull operations in the WBEM server. After the first call
        to an `Iter...()` method, the variable will be either `True` or `False`.
```